microsoft-edge/devtools-guide-chromium/whats-new/2021/07/devtools.md

```diff
@@ -206,7 +206,7 @@ If you are on Windows, Linux, or macOS, consider using the [Microsoft Edge previ
 
 
 <!-- links -->
-[CustomizeDarkTheme]: ../../../customize/dark-theme.md "Apply color themes to Microsoft Edge DevTools | Microsoft Docs"
+[CustomizeDarkTheme]: ../../../customize/dark-theme.md "Apply color themes to DevTools | Microsoft Docs"
```

Comment:
We should now link to the new `../../../customize/theme.md` article instead of `../../../customize/dark-theme.md` since the PR where I renamed the file was merged.

Reply:
Good catch, don't know why no broken link was reported, maybe b/c the redirect file.  This is (was) the only instance of "dark-theme.md" in the repo in this branch other than the redirect file.